includes/services/extended/instagram-basic-display.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * Instagram service definition for Keyring.
+ * http://instagram.com/developer/
+ */
+
+class Keyring_Service_Instagram_Basic_Display extends Keyring_Service_OAuth2 {
+	const NAME  = 'instagram_basic_display';
+	const LABEL = 'Instagram Basic Display';
+
+	function __construct() {
+		parent::__construct();
+
+		// Enable "basic" UI for entering key/secret
+		if ( ! KEYRING__HEADLESS_MODE ) {
+			add_action( 'keyring_' . $this->get_name() . '_manage_ui', array( $this, 'basic_ui' ) );
+			add_filter( 'keyring_i' . $this->get_name() . '_basic_ui_intro', array( $this, 'basic_ui_intro' ) );
```

Comment:
Looks like a stray `i` here at the start of the filter name.

Reply:
fixed